mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -844,6 +865,10 @@ CLASS net/minecraft/unmapped/C_astfners net/minecraft/entity/Entity
 	METHOD m_ynhqjslw getX ()D
 	METHOD m_ynqvoxet getAttachmentPos ()Lnet/minecraft/unmapped/C_vgpupfxx;
 	METHOD m_yoauovfn playExtinguishSound ()V
+	METHOD m_yontuqgh isCloserThan (Lnet/minecraft/unmapped/C_astfners;DD)Z
```

Comment:
Is closer than to what? Hmmm

Reply:
This one is a bit odd

Reply:
The yarn is `isInRange`. So could use that or look for something else

Reply:
```suggestion
	METHOD m_yontuqgh isInRange (Lnet/minecraft/unmapped/C_astfners;DD)Z
```